sys-fs/btrfsmaintenance/btrfsmaintenance-0.4.ebuild

```diff
@@ -0,0 +1,63 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+inherit systemd
+
+DESCRIPTION="Scripts for btrfs maintenance tasks like periodic scrub, balance, trim or defrag"
+HOMEPAGE="https://github.com/kdave/btrfsmaintenance"
+SRC_URI="https://github.com/kdave/btrfsmaintenance/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="systemd"
+
+DEPEND=""
+RDEPEND="${DEPEND}
```

Comment:
Delete empty variables.

Reply:
it's a used pattern on whole portage tree. I prefer to keep this (and the 0.3.1 package already does)

Reply:
That may be, but we can do better can't we?

Reply:
to be honest, I don't share your point. BTW I will remove DEPEND.

Reply:
Thank you.

What's really more common is to include RDEPEND in DEPEND:

    RDEPEND="some things"
    DEPEND="${RDEPEND}"

More often than not, run time dependencies are build time dependencies instead of the other way around. Leaving an empty DEPEND and including it in RDEPEND sets up the potential to incorrectly include build-time-only dependencies as run time dependency.